tdesign/src/Icon/cssSvgStrings.ts

```diff
@@ -28,6 +28,8 @@ export const apiIconURI =
   "data:image/svg+xml;charset=utf8,%3Csvg width='24' height='24' viewBox='0 0 24 24' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M13.5 3L10 20' stroke='black'/%3E%3Cpath d='M7.89996 8L3.49996 12L7.89996 16' stroke='%23000202' stroke-linejoin='round'/%3E%3Cpath d='M16 8L20.4 12L16 16' stroke='%23000202' stroke-linejoin='round'/%3E%3C/svg%3E";
 export const arrowRightURI =
   "data:image/svg+xml;charset=utf8,%3Csvg width='10' height='10' viewBox='0 0 10 10' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M7.4 5L7.90451 5.55495C8.06086 5.41281 8.15 5.21131 8.15 5C8.15 4.78869 8.06086 4.58719 7.90451 4.44504L7.4 5ZM3.5045 9.55496L7.90451 5.55495L6.8955 4.44504L2.4955 8.44504L3.5045 9.55496ZM7.90451 4.44504L3.5045 0.445044L2.49549 1.55495L6.8955 5.55495L7.90451 4.44504ZM7.4 4.25L-1.59301e-07 4.25L-1.9039e-07 5.75L7.4 5.75L7.4 4.25Z' fill='%23F94569'/%3E%3C/svg%3E";
+export const arrowRightURI2 =
```

Comment:
Is there some other name we can use than just appending `2` to it :smile: ? Something that would better describe how this one differs from `arrowRight`.

Reply:
btw. the name should probably be

```suggestion
export const arrowRight2URI =
```

because it's the URI for the 2nd arrow, not the 2nd URI for the one arrow

Reply:
I had to create another icon because I could not get a few things working correctly with the first one. Something to do with how the dimensions and color are hardcoded in the URI IIRC. Hopefully using an Icon component could mitigate and we can get rid of `ArrowRight2URI`